odmltables/odml_table.py

```diff
@@ -971,7 +972,15 @@ def _convert_single_value(self, value, dtype):
         elif dtype == 'float':
             result = float(value)
         elif dtype == 'boolean':
-            result = bool(value)
+            # strip whitespace and convert to lower case
+            value = value.strip().lower()
+            if value in ('y', 'yes', 't', 'true', 'on', '1'):
+                result = True
+            elif value in ('n', 'no', 'f', 'false', 'off', '0'):
+                result = False
+            else:
+                warnings.warn("invalid truth value %r, replacing with None" % (value,))
+                result = None
```

Comment:
I think this is a sufficient test for a good truth value. Sadly, Excel also has a tendency to output true/false in its native langue (e.g., wahr/falsch), but this cannot be captured. I think the warning is sufficient.